_pages/kriskirov.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: kriskirov
+pageUrl: kriskirov.github.io
+timestamp: TODAYS-24-2017-04-Mo
+---
```

Comment:
I can't see any notifications on the review You make. Is it by design? Is it intended?